smoketest/graphqloncall_test.go

```diff
@@ -1,11 +1,16 @@
 package smoketest
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
-	"github.com/target/goalert/smoketest/harness"
-	"strings"
+	"html/template"
```

Comment:
Should be `text/template` otherwise, it will interpret and try to escape values as an HTML doc.
```suggestion
	"text/template"
```